consensus/ising/proposerservice.go

```diff
@@ -14,17 +14,16 @@ import (
 	"github.com/nknorg/nkn/events"
 	"github.com/nknorg/nkn/net/message"
 	"github.com/nknorg/nkn/net/protocol"
+	"github.com/nknorg/nkn/util/config"
 	"github.com/nknorg/nkn/util/log"
 	"github.com/nknorg/nkn/vault"
 )
 
 const (
 	TxnAmountToBePackaged      = 1024
-	ConsensusTime              = 10 * time.Second
 	WaitingForFloodingFinished = time.Second * 1
```

Comment:
If we move ConsensusTime to config, probably we should move all these constants to config also?

Reply:
Moving the important time ConsensusTime and ProposerChangeTime to config since other modules may depend on these parameter, e.g.:block proposer verification when timeouts.
Other constants can continue to be placed in the consensus module.